client/hotrod-client/src/test/java/org/infinispan/client/hotrod/ExecTest.java

```diff
@@ -103,6 +117,64 @@ public void testRemoteMapReduce() throws Exception {
       assertTrue(results.get("macbeth").equals(Double.valueOf(287)));
    }
 
+   public void testRemoteMapReduceWithStreams() throws Exception {
+      String cacheName = "testRemoteMapReduce_Streams";
+      ConfigurationBuilder builder = getDefaultClusteredCacheConfig(CacheMode.DIST_SYNC, true);
+      builder.dataContainer().keyEquivalence(new AnyServerEquivalence()).valueEquivalence(new AnyServerEquivalence()).compatibility().enable().marshaller(new GenericJBossMarshaller());
+      defineInAll(cacheName, builder);
+      RemoteCache<String, String> cache = clients.get(0).getCache(cacheName);
+      RemoteCache<String, String> scriptCache = clients.get(0).getCache(SCRIPT_CACHE);
+      loadData(cache, "/macbeth.txt");
+      loadScript(scriptCache, "/wordCountStream.js");
+
+      HashMap<String, Long> results = cache.execute("wordCountStream.js", new HashMap<String, String>());
+      assertEquals(3209, results.size());
+      assertTrue(results.get("macbeth").equals(Long.valueOf(287)));
+   }
+
+   public void testRemoteMapReduceWithStreams_DistributedMode() throws Exception {
+      String cacheName = "testRemoteMapReduce_Streams_dist";
+      ConfigurationBuilder builder = getDefaultClusteredCacheConfig(CacheMode.REPL_SYNC, true);
+      builder.dataContainer().keyEquivalence(new AnyServerEquivalence()).valueEquivalence(new AnyServerEquivalence())
+              .compatibility().enable().marshaller(new GenericJBossMarshaller());
+      defineInAll(cacheName, builder);
+      waitForClusterToForm(cacheName);
+
+      RemoteCache<String, String> cache = clients.get(0).getCache(cacheName);
+      RemoteCache<String, String> scriptCache = clients.get(1).getCache(SCRIPT_CACHE);
+      loadData(cache, "/macbeth.txt");
+      loadScript(scriptCache, "/wordCountStream_dist.js");
+
+      ArrayList<HashMap<String, Long>> results = cache.execute("wordCountStream_dist.js", new HashMap<String, String>());
```

Comment:
The same here... 

There are more places like this in this commit. Could you please update them also?